sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -765,12 +839,24 @@ unsigned +
 The choice of a valid point for the start sequence is determined by the sequencing rule. +
 Format: n, n +
 Example: "0,0" (without quotes)
+
+| 10
+| Vertical datum
+| verticalDatum
+^| 0..1
+<| Integer +
+unsigned +
+16-bit
+| see remark <<tab-root-group-attributes>> +
+row vertical datum
 |===
 
 The gridOriginLongitude, gridOriginLatitude, gridSpacingLongitudinal, and gridSpacingLatitudinal attributes should be in the same geographic units as the bounding box. Note that this practice deviates from S-100 where it indicates that this value should be in Arc Degrees.
 
 numPointsLongitude and numPointsLatitude must contain the number of cells in the x and y dimensions of the values table.
 
+The S-102 uses the "Attribute Overwriting" option of the <<iho-s100,part=10c,clause=9.7.1>>. This allows the feature instance group to overwrite the attributes of a higher group, in this case the 'verticalDatum'. The default vertical datum is specified in the root group (see table <<tab-root-group-attributes>>). A new separate feature instance group must be created for all areas with a vertical datum that differs from the default vertical datum. Only feature instance groups with a value that differs from the default vertical datum use the additional attribute "verticalDatum" on the feature instance group. The use of multiple vertical datums within an S-102 product requires the use of "domainExtent.polygon". In addition, according to S-100, either the BoundingBox at the Feature Instance Group or the "domainExtent.polygon" must be specified. When using multiple vertical datums, the use of the "domainExtent.polygon" is mandatory, which means that the BoundingBox is not specified in this case. The grid cells that do not belong to the area of the respective vertical datum are to be assigned the no-data values.
+
```

Comment:
I suggest making the part about domainExtent.polygon a stand-alone paragraph for better readability.

Furthermore: 

- Don't we risk that the domainExtent.polygon has a huge number of points in order to follow the grid coverage of a specific vertical datum? Is some sort of simplification/generalization of such polygons allowed? 
- Are the polygons allowed to overlap? A statement about this should be added for increased clarity.

Reply:
> I suggest making the part about domainExtent.polygon a stand-alone paragraph for better readability.

Good point, thank you.

---
> * Don't we risk that the domainExtent.polygon has a huge number of points in order to follow the grid coverage of a specific vertical datum?

Unfortunately, yes. But this is the way that the S-100 5.2.0 provides for and is explicitly described in chapter 10c-9.7.1. In HDF5, however, you can use compression to counteract the increasing file size.

---
> * Is some sort of simplification/generalization of such polygons allowed?

Yes, partly. The edges between the VerticalDatum areas must be exact (According to S-101 DEPARE & S-98). You can of course generalize in the peripheral areas. The polygon does not have to follow the boundaries of the grid cells.
This is how it looks for us. Our peripheral areas are also only roughly included, there is usually still land somewhere anyway. 
![VDatum_BSH_Example](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/ea1eb155-f955-4a41-baa6-128c2c4a3f68)

---
> * Are the polygons allowed to overlap?

No, at no point.

---
> * A statement about this should be added for increased clarity.

Okay, we'll try to find a suitable location.

Reply:
If the QualityofBathymetryCoverage is shared and there is no overlap, why don't add a vertical datum attribute in it?
This way we don't need to have multiple domainExtent.polygon instances.

In other words, if there are multiple vertical datums, then it becomes mandatory to populate a vertical datum attribute in the QualityofBathymetryCoverage.

Reply:
The enclosing polygon can be dissected into one or more simple polygons, with a separate feature instance for each.

Reply:
Because that would unfortunately lead to incorrect data.
In reality, the boundary between the different VerticalDatums does not follow the boundaries of the grid cells.
This means that a grid cell can lie in several VerticalDatum areas.
However, the grid cell cannot have several depth values. We have therefore introduced several coverages that can describe the same grid cell.
However, the fact that the grid cell belongs to several VerticalDatum areas does not change the metadata of the grid cell, as described in QualityOfBathymetryCoverage.
For example, the survey date (start/end) remains the same.
Only the conversion of the depth value changes. This is mapped in different BathymetryCoverage.

Unfortunately, the coarser the resolution of the grid, the greater the problem that arises.
The following images show a corresponding example.
You can see the lock in Kiel (entrance to the Kiel Canal) (as ENC in the background).
The lock gate is recognizable.
The green line is the boundary of the VerticalDatum areas, created using the S-57 M_SDAT area.
In the Kiel Canal, the depths refer to "LocalDatum" in the Baltic Sea to "BSCD2000". This means that the indentical measuring point (grid cell) differs by approx. 0.2 m in both reference systems.
The red squares are the grid cells from the S-102 at different resolutions (1 m, 5 m, 10 m, 15 m).
It is clear that the coarser the resolution of the grid, the greater the effect of the vertical date limit on the depths.
With the 10 m resolution, there would already be areas on one side or the other of the sluice gate where the wrong depth is in the grid cell.
The solution of writing the vertical datum to the FeatureAttribute table is therefore not possible if surveying is carried out over a vertical datum limit (which is the case here).
Writing the VerticalDatum to the grid cell in the QualityOfBathymetryCoverage does not work either.
This is not provided for in the S-100 and the DataCodingFormat 9. In addition, the information as described above would not be sufficient.

1m GridSpacing
![GridSpacing_1m](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/eeea9fae-0aba-4597-9b1e-5db5b9a5d771)

5m GridSpacing
![GridSpacing_5m](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/cb589289-54d9-4073-abe2-7ceb2809e323)

10m GridSpacing
![GridSpacing_10m](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/c7e9dc9f-2d58-4b53-a545-fbc45732d597)

15m GridSpacing
![GridSpacing_15m](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/f009505f-6803-447d-b01f-145102dc1239)

Reply:
Separation of the domainExtent.polygon paragraph done in commit 
84774e9a7ba2a211552feee2275d9ab13e4838b3

Reply:
> > If the QualityofBathymetryCoverage is shared and there is no overlap, why don't we add a vertical datum attribute in it?
> 
> Because that would unfortunately lead to incorrect data. In reality, the boundary between the different VerticalDatums does not follow the boundaries of the grid cells. This means that a grid cell can lie in several VerticalDatum areas. However, the grid cell cannot have several depth values. We have therefore introduced several coverages that can describe the same grid cell. However, the fact that the grid cell belongs to several VerticalDatum areas does not change the metadata of the grid cell, as described in QualityOfBathymetryCoverage. For example, the survey date (start/end) remains the same. Only the conversion of the depth value changes. This is mapped in different BathymetryCoverage.
> 
> Unfortunately, the coarser the resolution of the grid, the greater the problem that arises. The following images show a corresponding example. You can see the lock in Kiel (entrance to the Kiel Canal) (as ENC in the background). The lock gate is recognizable. The green line is the boundary of the VerticalDatum areas, created using the S-57 M_SDAT area. In the Kiel Canal, the depths refer to "LocalDatum" in the Baltic Sea to "BSCD2000". This means that the indentical measuring point (grid cell) differs by approx. 0.2 m in both reference systems. The red squares are the grid cells from the S-102 at different resolutions (1 m, 5 m, 10 m, 15 m). It is clear that the coarser the resolution of the grid, the greater the effect of the vertical date limit on the depths. With the 10 m resolution, there would already be areas on one side or the other of the sluice gate where the wrong depth is in the grid cell. The solution of writing the vertical datum to the FeatureAttribute table is therefore not possible if surveying is carried out over a vertical datum limit (which is the case here). Writing the VerticalDatum to the grid cell in the QualityOfBathymetryCoverage does not work either. This is not provided for in the S-100 and the DataCodingFormat 9. In addition, the information as described above would not be sufficient.
> 
> 1m GridSpacing ![GridSpacing_1m](https://private-user-images.githubusercontent.com/85698889/335496962-eeea9fae-0aba-4597-9b1e-5db5b9a5d771.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MjYzNDAsIm5iZiI6MTcxNzUyNjA0MCwicGF0aCI6Ii84NTY5ODg4OS8zMzU0OTY5NjItZWVlYTlmYWUtMGFiYS00NTk3LTliMWUtNWRiNWI5YTVkNzcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE4MzQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiY2M4NjExN2M1NDc2NmZhMGY4MjM4M2Y3ODgwMDc5ZDc4ZjE1MjA0ZjIzN2I2OGU4ZTk5NjliZjU1NGJlMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0Ogh3yMVGW2Mia5Jz4gFP0xrWyAgFQn5buuSlrwW--g)
> 
> 5m GridSpacing ![GridSpacing_5m](https://private-user-images.githubusercontent.com/85698889/335497023-cb589289-54d9-4073-abe2-7ceb2809e323.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MjYzNDAsIm5iZiI6MTcxNzUyNjA0MCwicGF0aCI6Ii84NTY5ODg4OS8zMzU0OTcwMjMtY2I1ODkyODktNTRkOS00MDczLWFiZTItN2NlYjI4MDllMzIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE4MzQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZjZjgyNDRiZGY2OGIzMjg4ZGQyZWMzMTk5MTlmMTNlOTg5MDljMTZhNjJkNjMwMWYwMGUwZjZkN2UyNTUyZTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dNTXK4XtoaxEn_u7F8NdPomLfZEgJGLDTB0ovOhTcOc)
> 
> 10m GridSpacing ![GridSpacing_10m](https://private-user-images.githubusercontent.com/85698889/335497099-c7e9dc9f-2d58-4b53-a545-fbc45732d597.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MjYzNDAsIm5iZiI6MTcxNzUyNjA0MCwicGF0aCI6Ii84NTY5ODg4OS8zMzU0OTcwOTktYzdlOWRjOWYtMmQ1OC00YjUzLWE1NDUtZmJjNDU3MzJkNTk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE4MzQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3Y2M2Y2I2ZmYyMWY2YjY5YWE3NTdmNDA4ZWFkMmNlNWMzY2QwZTliZjIzMjQ0MTRlNDVkMWU5MjM0NmYwNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8wOQ9TYMu51yzlgDXCMo92z0YANKo7t6-PcfDLZx8Xs)
> 
> 15m GridSpacing ![GridSpacing_15m](https://private-user-images.githubusercontent.com/85698889/335499941-f009505f-6803-447d-b01f-145102dc1239.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MjYzNDAsIm5iZiI6MTcxNzUyNjA0MCwicGF0aCI6Ii84NTY5ODg4OS8zMzU0OTk5NDEtZjAwOTUwNWYtNjgwMy00NDdkLWIwMWYtMTQ1MTAyZGMxMjM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDE4MzQwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNiNGJiM2IzOGQ2YWFhNjQwODFjOGJkNGY4ZjYxMjkwNmFhMzllZWZmNmJkNGRiYzE5MzlhYTdkNTk5MmViYTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pL3wQLCHQ9B2lyHRXOO2wGD4vcYixIFq1rhRbRh-7UY)

I believe that this issue is related to the lack of use of the [display scale parameters](https://github.com/iho-ohi/S-102-Product-Specification/discussions/85). Should the grid resolution being a function of the ENCs that will be visible at the display scale range?